metrics/run_all_metrics.sh

```diff
@@ -17,6 +17,14 @@
 set -e
 
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/lib/common.bash"
+
+# Check arguments
+if [[ ! -v RUNTIME ]]; then
+    die Variable RUNTIME must be set to the name of your CC runtime
```

Comment:
nitpick: it doesn't have to be a CC runtime ;-) But, seeing as this is a CC repo, it's not really a problem, and won't stop merge :-)